### ISSUE TITLE
Not working for Ruby 3.0

### ISSUE THREAD

**Issue by acallaghan** [square/square-ruby-sdk#64]
Hi there

I've noticed that the method patterns used in this repo will only work for Ruby 2.x for many endpoints. The minimum failure case for me is this:

```
require 'square'

# Initialize Square Client
square = Square::Client.new(
  access_token: 'EAAAE...'
)

result = square.devices.list_device_codes

if result.success?
    puts result.data
elsif result.error?
    warn result.errors
end
```

Running this returns
```
~/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/square.rb-9.0.0.20210226/lib/square/api_helper.rb:67:in `append_url_with_query_parameters': wrong number of arguments (given 1, expected 2) (ArgumentError)
	from ~/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/square.rb-9.0.0.20210226/lib/square/api/devices_api.rb:30:in `list_device_codes'
	from square-test.rb:10:in `<main>'
```

Are there any plans to support Ruby 3.0?

**Comment by deanpapastrat** [comment]
Thanks for bringing it up! Ruby 3.0 support is in the works - we definitely plan to support it, but it requires some upstream changes that we're making right now.

**Comment by acallaghan** [comment]
Ok no worries, that's great thank you

**Comment by hukid** [comment]
@acallaghan we just released [9.1.1.20210317](https://github.com/square/square-ruby-sdk/releases/tag/9.1.1.20210317) which support Ruby 3.0. Please let us know if that solves your problem.

**Comment by acallaghan** [comment]
That's great thanks, I'll check it out soon
